### ISSUE TITLE
Reports about excessive logging - tweak log levels?

### ISSUE THREAD

**Issue by rjan90** [filecoin-project/go-f3#770]
Got some user reports about these log lines:

1: https://github.com/filecoin-project/go-f3/blob/12514c13d20f9e08597e2aac82e8d58fff5990e3/host.go#L414-L415
2. https://github.com/filecoin-project/go-f3/blob/12514c13d20f9e08597e2aac82e8d58fff5990e3/host.go#L195-L196

I feel like 1 is okay to drop down to DEBUG. But I´m uncertain about 2?

**Comment by masih** [comment]
About 2, do you have the actual log line that folks are getting?

**Comment by rjan90** [comment]
> About 2, do you have the actual log line that folks are getting?

I only have this screenshot that I got: 
<img width="735" alt="Screenshot 2024-12-03 at 12 59 34" src="https://github.com/user-attachments/assets/58960edf-251b-414a-b5f2-6ef36b7be334">

**Comment by masih** [comment]
Thanks Orjan.  Two questions:
* How often does this error occur for that user?
* How popular is this error across users?

This is a serious error and suggests potentially corrupt or incomplete state.

**Comment by rjan90** [comment]
> This is a serious error and suggests potentially corrupt or incomplete state.

Its on their Calibration network node, which might explain why that user was seeing this error?

**Comment by masih** [comment]
Yes; on calibnet it is far more likely.  On mainnet if folks see this error we want to know about it 😊

**Comment by masih** [comment]
Goging back to what this issue is about, I would keep the ERROR log level for case No. 2. 

Having updated the log level for case No. 1. I think we can close this issue, unless there is anything else that I might have missed?
